eclipse/plugins/quarkus.properties

```diff
@@ -0,0 +1,3 @@
+plugin_url=https://download.jboss.org/jbosstools/photon/stable/composite/4.21.3
```

Comment:
shall we "hardcode" a specific version here?
How about using https://download.jboss.org/jbosstools/photon/stable/updates/ instead?
Slightly related to https://github.com/devonfw/ide/issues/832

Reply:
It would be nice if it was possible to install newer versions of quarkus plugin, but because the Eclipse IDE version is lower than the required version for the newest quarkus plugin, the dependencies can't be installed correctly and it gives an error. The issue you referenced in #809 might solve the problem, but I would need to test if that actually works

Reply:
The idea of using the latest stable updates site was correct. My Eclipse IDE version was not up to date, which is why the dependencies couldn't be installed. I am closing this conversation, as everything works correctly with the `plugin_url=https://download.jboss.org/jbosstools/photon/stable/updates/`